xcore/fisheye_image_file.cpp

```diff
@@ -162,7 +162,8 @@ FisheyeImageFile::read_roi (const SmartPtr<VideoBuffer> &buf, uint32_t idx)
             h += y_step;
         }
 
-        fseek (_fp, fp_offset, SEEK_CUR);
+        XCamReturn ret = seek (_fp, fp_offset, SEEK_CUR);
```

Comment:
**seek**
04:56:04,122 INFO  - fisheye_image_file.cpp: In member function ‘XCamReturn XCam::FisheyeImageFile::read_roi(const XCam::SmartPtr<XCam::VideoBuffer>&, uint32_t)’:
04:56:04,122 INFO  - fisheye_image_file.cpp:165:26: error: ‘seek’ was not declared in this scope; did you mean ‘fseek’?
04:56:04,122 INFO  -   165 |         XCamReturn ret = seek (_fp, fp_offset, SEEK_CUR);
04:56:04,122 INFO  -       |                          ^~~~
04:56:04,122 INFO  -       |                          fseek
04:56:04,123 INFO  - fisheye_image_file.cpp:166:2: error: ‘CHECK’ was not declared in this scope
04:56:04,123 INFO  -   166 |  CHECK (ret, "fseek failed.");
04:56:04,123 INFO  -       |  ^~~~~
04:56:04,137 INFO  - make[2]: *** [Makefile:847: libxcam_core_la-fisheye_image_file.lo] Error 1
04:56:04,137 INFO  - make[2]: *** Waiting for unfinished jobs....